third-party/test/index.ts

```diff
@@ -124,6 +124,12 @@ function run(testsRoot, clb): any {
     const testFile = process.env["TEST_FILE"] === 'null' ? undefined : process.env["TEST_FILE"]
     const testFilePath = testFile?.replace(/^src\/test\//, "")?.concat('.js')
 
+    const globalSetupPath = paths.join(testsRoot, 'globalSetup.test.js')
+    if (testFilePath && fs.existsSync(globalSetupPath)) {
+        // XXX: explicitly add globalSetup, other tests depend on it.
```

Comment:
What does the XXX mean?

Reply:
it is a convention that means "this is a hack"